src/Pages/instructor/manage-class/class-card.js

```diff
@@ -0,0 +1,79 @@
+
+import React from "react";
+import styled from "styled-components";
+import { Link } from "react-router-dom";
+import { HeroImgJr } from "../../../reusable-components/reusableComponents";
+
+const StyledClassImg = styled(HeroImgJr)`
+	border-radius: 25px 25px 0px 0px;
+	height: 50%;
+`;
+
+const Button = styled(Link)`
+	background-color: ${pr => pr.theme.primaryOrange};
+	color: ${pr => pr.theme.charcoal};
+	font-size: 2rem;
+	border-radius: 5px;
+    font-weight: 500;
+    width: 100%;
+    margin: 5px;
+    padding: 5px 20px;
+	display: flex;
+	flex-flow: row nowrap;
+	justify-content: center;
+`
+
+const StyledClassCard = styled.div`
+	width: 35rem;
+	height: 24rem;
+	margin: 2rem 2rem;
+	.title {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        height: 100%;
+		background: rgba(248, 248, 248, 0.2);	
+	}
+	.details {
+		display: flex;
+		flex-flow: column nowrap;
+		justify-content: space-evenly;
+		align-items: center;
+		width: 100%;
+		height: 50%;
+		background: transparent;
+		border-radius: 0px 0px 25px 25px;
+		border: 1px solid #CCCCCC;
+		outline: none;
+
+        .btns {
+            display: flex;
+        }
+
+	}
+`;
+
+
+const ClassCard = (props) => {
```

Comment:
nice reuse of available styled components, I really love that we've been recycling code so much. Such a good practice.